cypress/cypress.config.js

```diff
@@ -67,40 +65,7 @@ module.exports = {
           return global.specRunVersions.get(specRunName) || null;
         },
       });
-      on("before:browser:launch", (browser = {}, launchOptions = {}) => {
-        launchOptions.args = launchOptions.args.filter(
-          (item) => item !== "--disable-dev-shm-usage",
-        );
-        if (browser.name === "chrome" && browser.isHeadless) {
-          launchOptions.args.push("--window-size=1200,800");
-          launchOptions.args.push("--force-device-scale-factor=1");
-        }
-
-        if (browser.name === "electron" && browser.isHeadless) {
-          launchOptions.preferences.width = 1200;
-          launchOptions.preferences.height = 1800;
-        }
-
-        if (browser.name === "firefox" && browser.isHeadless) {
-          launchOptions.args.push("--width=1200");
-          launchOptions.args.push("--height=800");
-        }
-
-        return launchOptions;
-      });
-      on("after:spec", (spec, results) => {
-        if (results && results.video) {
```

Comment:
This was added as a workaround for video deletion on pass param that got deprecated, but since we're removing videos entirely just going to delete this snippet in case it causes issues